templates/exim.conf.j2

```diff
@@ -0,0 +1,63 @@
+# {{ ansible_managed }}
+#
+
+######################################################################
+#                     LOCAL VARIABLE SETTINGS                        #
+######################################################################
+
+{{ exim_local_section }}
+
+######################################################################
+#                     MAIN CONFIGURATION SETTINGS                    #
+######################################################################
+
+{{ exim_main_section }}
+
+######################################################################
+#                     ACL CONFIGURATION                              #
+######################################################################
+
+begin acl
+
+{{ exim_acls_section | default("") | indent( width=8, indentfirst=True) }}
+
+######################################################################
+#                     ROUTERS CONFIGURATION                          #
+######################################################################
+
+begin routers
+
+{{ exim_router_section | default("") | indent( width=8, indentfirst=True) }}
+
+######################################################################
+#                     TRANSPORTS CONFIGURATION                       #
+######################################################################
+
+begin transports
+
+{{ exim_transport_section | default("") | indent( width=8, indentfirst=True) }}
+
+######################################################################
+#                     AUTHENTICATORS CONFIGURATION                   #
+######################################################################
+
+begin authenticators
+
+{{ exim_authenticator_section | default("") | indent( width=8, indentfirst=True) }}
```

Comment:
You could mention in your readme that you expect a (multi-line) string for each section.

Reply:
fixed